CV/Official_Announcement.md

```diff
@@ -1,9 +1,9 @@
 
-Để giúp một số thành viên của hội nắm bắt được kiến thức căn bản về AI/ML, đặc biệt là các bạn đang làm Software Engineer muốn học và tìm hiểu về lĩnh vực này, từ năm 2019, hội VJAI sẽ tổ chức thêm 1 hoạt động mới là các khoá học ngắn về từng chủ đề trong lĩnh vực AI. Chúng ta sẽ bắt đầu với khoá học về chủ đề `Computer Vision and Deep Learning for beginner`.
```

Comment:
Có cần chỉnh tên course thành "Deep Learning in Computer Vision for Beginners" cho giống phần mục đích?